src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue24878.cs

```diff
@@ -13,6 +13,7 @@ public Issue24878(TestDevice device) : base(device) { }
 
 	[Test]
 	[Category(UITestCategories.ToolbarItem)]
+	[FailsOnWindows("ToolbarItem changes text color on theme change, but support for icon color is not implemented")]
```

Comment:
This attribute has changed, should use a compilation directive to avoid Windows.

Reply:
`D:\a\_work\1\s\src\Controls\tests\TestCases.Shared.Tests\Tests\Issues\Issue24878.cs(16,3): error CS0246: The type or namespace name 'FailsOnWindows' could not be found (are you missing a using directive or an assembly reference?) [D:\a\_work\1\s\src\Controls\tests\TestCases.WinUI.Tests\Controls.TestCases.WinUI.Tests.csproj]
`

Reply:
Thanks! done :)